MODULE.bazel

```diff
@@ -0,0 +1,112 @@
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_python", version = "0.31.0")
+bazel_dep(name = "rules_kotlin", version = "1.9.1")
+bazel_dep(name = "rules_java", version = "7.4.0")
+bazel_dep(name = "rules_jvm_external", version = "6.0")
+
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+maven.install(
+    artifacts = [
+        # smithy
+        "software.amazon.smithy:smithy-model:1.45.0",
+        "software.amazon.smithy:smithy-codegen-core:1.45.0",
+        "software.amazon.smithy:smithy-utils:1.45.0",
+
+        # tests
+        "org.junit.jupiter:junit-jupiter:5.10.2",
+        "org.junit.jupiter:junit-jupiter-api:5.10.2",
+        "org.junit.jupiter:junit-jupiter-engine:5.10.2",
+        "org.junit.jupiter:junit-jupiter-params:5.10.2",
+        "org.junit.platform:junit-platform-console:1.10.2",
+
+        # docs
+        "org.scalameta:mdoc_2.13:2.5.2",
+
+        # formatting
+        "org.scalameta:scalafmt-cli_2.13:3.8.0",
+
+        # scala
+        "org.scala-lang.modules:scala-collection-compat_2.13:2.11.0",
```

Comment:
If we don't compile to `2.12` anymore we should not need this `compat` dependency. We can try to remove it in this PR or another one.

Reply:
Will Bazel be able to cross-compile to Scala 2.13 and Scala 3?

Reply:
> Will Bazel be able to cross-compile to Scala 2.13 and Scala 3?
Yes, although [*will* is the key word here](https://github.com/bazelbuild/rules_scala/pull/1552). I mean, it should be possible now, but [it's cumbersome](https://eed3si9n.com/cross-build-anything-with-bazel/).

With the current setup we don't support Scala 3 anyway (or rather - support for it is not turned on by default, because I guess sbt supports cross building 2.13 and 3?)